database/csvSeed.js

```diff
@@ -0,0 +1,102 @@
+const fs = require('fs');
+const csvWriter = require('csv-write-stream');
```

Comment:
Refactor idea: remove this csvWriter npm package and use pure node capabilities.

Reply:
Thank you. I refactor the code and try to improve things, but it still takes a lot of time not sure why. I will try to check with a TM.